sys/fuchsia/fidlgen/main.go

```diff
@@ -69,6 +76,41 @@ func main() {
 		if err != nil {
 			failf("fidlgen failed: %v\n", err)
 		}
+
+		newFiles = append(newFiles, fmt.Sprintf("%s.txt", txtPath))
+	}
+
+	var unused []ast.Pos
+	desc := ast.ParseGlob("*.txt", nil)
+	if desc != nil {
```

Comment:
failf here if parsing fails (it should not, right?)

Reply:
It should not; fidlgen generates syntactically correct description. Done.